pkg/util/rowcodec/common.go

```diff
@@ -264,6 +265,9 @@ type RowData struct {
 	// Data stores the result of Encode. However, it mostly acts as a buffer for encoding columns on checksum
 	// calculation.
 	Data []byte
+
+	// loc is used to convert Timestamp value to UTC for checksum calculation.
+	Loc *time.Location
```

Comment:
Instead of adding a new field here, why not to let `Checksum` accept a `loc` argument? (just like `Encode`)